src/main/java/com/dev/cinema/model/Order.java

```diff
@@ -0,0 +1,69 @@
+package com.dev.cinema.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.MapsId;
+import javax.persistence.OneToMany;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "orders")
+public class Order {
+    @Id
+    private Long id;
+    @OneToMany
+    @JoinTable(
+            name = "order_ticket",
+            joinColumns = @JoinColumn(name = "order_id"),
+            inverseJoinColumns = @JoinColumn(name = "ticket_id")
+    )
+    private List<Ticket> tickets;
+    private LocalDateTime orderDate;
+    @OneToOne
```

Comment:
```suggestion
    @ManyToOne
```
User can have a few orders

Reply:
I agree. Fixed.